Automattic/woocommerce-payments
# Move the applyThousandSeparator method to utils

Fixes #4566

#### Changes proposed in this Pull Request

In #4527, we used the `applyThousandSeparator` method to add a thousand separator to the transactions number. Later we found out that we need to add the separator to the deposits and disputes too.

This PR moves the `applyThousandSeparator` method to `client/utils/index.js` and then imports the method to use in the transactions, deposits, and disputes counts. This will keep the logic separate and we will also be able to add tests to the method.









#### Testing instructions

1. Clone the `update/move-applyThousandSeparator-method-to-utils` branch in your local machine
2. Navigate to `woocommerce-payments/client/transactions/list/index.tsx`
3. Run `npm run watch` in the terminal
4. Find this line in the index.tsx file: `applyThousandSeparator(transactionsSummary.count as number)`
5. Change `transactionsSummary.count` to any number greater than 999 and save changes
6. Refresh `wp-admin > Payments > Transactions` and you should see the thousand separator in the transaction count.
7. Deposits can be tested in the similar way from `woocommerce-payments/client/deposits/list/index.tsx`
8. And disputes can be tested in the similar way from `woocommerce-payments/client/disputes/index.tsx`





-------------------


**Post merge**
